### DISCUSSION TITLE
v3.1.4

### DISCUSSION THREAD

**Discussion by InfinityLoop1308** [InfinityLoop1308/PipePipe#150] [Announcements]
### New Features:

- Allow migration from NewPipe 0.25.0+ 

### Bug Fixes:

- 'Account' Icon Incorrect Colour with Dark Theme Enabled [GitHub #142]
- Icon of bullet comment status is not showed correctly
- Unable to view multi-p video (BiliBili) [GitHub #148]

<hr/>

### Which APK should you download?

- arm64-v8a should work on most devices.
- armeabi-v7a if your device is a 10+ year old phone.
- x86_64 for 64-bit Android simulators (on a PC).
- x86 for 32-bit simulators.

If these rules not working, download the universal APK. It should work on any device.
All APKs provide the same contents and functions, but the universal one may be slower than the split ones.

<hr /><em>This discussion was created from the release <a href='https://github.com/InfinityLoop1308/PipePipe/releases/tag/v3.1.4'>v3.1.4</a>.</em>

**Comment by RE1AX1** [comment]
Very good app, but would like some changes (YouTube only)
1. Make a version of the application only for YouTube client to reduce the weight of the application, 32 mb is a lot.
2. Optimize the application, as sometimes there are some lags (scrolling, transitions)
3. Long video downloads and the start of the application itself is slow (again, it would not hurt to make a YouTube-only version).
4. Make the sorting of videos on the channel "popular", "new", "old"
5. Make it possible to see new comments on videos.
